data/content/environment_variables.yaml

```
@@ -158,6 +158,11 @@ variables:
     values:
       - true
       - false
+  - name: BUILDKITE_CLUSTER_ID
+    desc: |
+      The UUID value of the cluster, but only if the build has an associated `cluster_queue`; otherwise, the env var is not set.
```

Thanks for your contribution @pzeballos . Perhaps spell out 'env var' in full...

```suggestion
      The UUID value of the cluster, but only if the build has an associated `cluster_queue`. Otherwise, this environment variable is not set.
```